sample-apps/springboot/src/main/java/com/amazon/sampleapp/MetricEmitter.java

```diff
@@ -65,95 +62,83 @@ public MetricEmitter() {
         meter
-            .longUpDownCounterBuilder(queueSizeChangeMetricName)
+            .upDownCounterBuilder(queueSizeChangeMetricName)
             .setDescription("Queue Size change")
             .setUnit("one")
             .build();
 
-    totalBytesSentObserver =
-        meter
-            .longSumObserverBuilder(totalApiBytesSentMetricName)
-            .setDescription("Total API request load sent in bytes")
-            .setUnit("one")
-            .setUpdater(
-                longResult -> {
-                  System.out.println(
-                      "emit total http request size "
-                          + totalBytesSent
-                          + " byte, "
-                          + apiNameValue
-                          + ","
-                          + statusCodeValue);
-                  longResult.observe(
-                      totalBytesSent,
-                      Labels.of(
-                          DIMENSION_API_NAME,
-                          apiNameValue,
-                          DIMENSION_STATUS_CODE,
-                          statusCodeValue));
-                })
-            .build();
-
-    apiLastLatencyObserver =
-        meter
-            .longValueObserverBuilder(lastLatencyMetricName)
-            .setDescription("The last API latency observed at collection interval")
-            .setUnit("ms")
-            .setUpdater(
-                longResult -> {
-                  System.out.println(
-                      "emit last api latency "
-                          + apiLastLatency
-                          + ","
-                          + apiNameValue
-                          + ","
-                          + statusCodeValue);
-                  longResult.observe(
-                      apiLastLatency,
-                      Labels.of(
-                          DIMENSION_API_NAME,
-                          apiNameValue,
-                          DIMENSION_STATUS_CODE,
-                          statusCodeValue));
-                })
-            .build();
-    actualQueueSizeObserver =
-        meter
-            .longUpDownSumObserverBuilder(actualQueueSizeMetricName)
-            .setDescription("The actual queue size observed at collection interval")
-            .setUnit("one")
-            .setUpdater(
-                longResult -> {
-                  System.out.println(
-                      "emit actual queue size "
-                          + actualQueueSize
-                          + ","
-                          + apiNameValue
-                          + ","
-                          + statusCodeValue);
-                  longResult.observe(
-                      actualQueueSize,
-                      Labels.of(
-                          DIMENSION_API_NAME,
-                          apiNameValue,
-                          DIMENSION_STATUS_CODE,
-                          statusCodeValue));
-                })
-            .build();
+    meter
```

Comment:
If we were to add a third sample app, we could probably extract these meter builders into a "commons" package or something

Reply:
Yeah this has been a great thing to do for a while :) This class even shows up in the aws-otel-test-framework repo :O